openfe/utils/visualization_3D.py

```diff
@@ -107,6 +107,45 @@ def _add_spheres(view:py3Dmol.view, mol1:Chem.Mol, mol2:Chem.Mol, mapping:Dict[i
         )
 
 
+@requires_package("py3Dmol")
+def show_component_coords(mols: Iterable[ExplicitMoleculeComponent],
+                     style: Optional[str] ="stick",
+                     shift: Optional[Tuple[float, float, float]] = None,
+                     ) -> py3Dmol.view:
+    """this function can be used to visualize multiple component coordinates in one view.
+    It helps to understand how the components are aligned in the system to each other.
+
+    Parameters
+    ----------
+    mols : Iterable[ExplicitMoleculeComponent]
+        collection of components
+    style : Optional[str], optional
+        py3Dmol style, by default "stick"
+    shift : Tuple of floats, optional
+        Amount to i*shift each mols_i in order to allow inspection of them in heavy overlap cases.
+        
```

Comment:
is it worth having the view as an optional argument so that existing views can be used?